handler.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const AWS = require('aws-sdk');
+const ccxt = require('ccxt');
+const axios = require('axios');
+
+module.exports.transaction = async (event) => {
+  var jobId = event.Records[0].Sns.Message;
+
+  var jobURL = `http://jobs.service.thinblock.io/jobs/${jobId}?look_up=_id`;
```

Comment:
Can we take this from env variable?
```suggestion
  const jobServiceBase = process.env.JOBS_SERVICE_BASE_URL || 'http://jobs.service.thinblock.io';
  const jobsUrl = `${jobServiceBase}/jobs/${jobId}?look_up=_id`
```

Reply:
Looks good. I'll make these changes.